config/config.go

```diff
@@ -57,6 +58,18 @@ func (config *Config) Validate(path string) ([]string, error) {
 				return nil, errors.New("cannot specify camera[data_frequency_hz] less than zero")
 			}
 		}
+
+		mapRateSec, ok := config.Camera["map_rate_sec"]
```

Comment:
Remind me again why we are adding this back? Was it removed by mistake?

Reply:
I think it was removed by mistake. It was not removed as part of the [scope](https://docs.google.com/document/d/1r-DCnFLFBG7jSZDcKmoIpVaR7IEe1YaD1qiqVeNbpj8/edit) and caused a test failure when I made the tests use the same helper function